node/runtime/src/lib.rs

```diff
@@ -58,8 +58,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("node"),
 	impl_name: create_runtime_str!("substrate-node"),
 	authoring_version: 10,
-	spec_version: 97,
-	impl_version: 99,
+	spec_version: 98,
+	impl_version: 100,
```

Comment:
isn't impl_version bumped to spec_version when spec_version is increased ?

Reply:
I am not sure, I was assuming that they are just bumped every time its required.

Reply:
yeah, impl version becomes spec version when spec version is bumped. it could also become 0, but i find that harder to parse.